recipes/spython/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "spython" %}
+{% set version = "0.0.51" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9cef4c79f2cb1fcafa4555f8abd4ccfbd7d2ef3081a9b526beb027b5ff52fe77
+
+build:
+  number: 0
+  skip: True  # [py2k]
```

Can you make this a noarch package and change the requirements to `python >=3`?
```suggestion
  noarch: python
```
([Though the `setup.py` suggests Python 2 should be supported](https://github.com/singularityhub/singularity-cli/blob/master/setup.py#L106))

---

Fixed!

The documentation [mentions](https://singularityhub.github.io/singularity-cli/install) that python3 is required.